VimeoNetworking/Sources/Models/SubscriptionCollection.swift

```diff
@@ -14,13 +14,14 @@ public class SubscriptionCollection: VIMModelObject
     
     public var uri: String?
     public var modifiedTime: NSDate?
-    public var subscriptions: Subscription?
+    public var subscription: Subscription?
     
     // MARK: - VIMMappable
     
     public override func getObjectMapping() -> AnyObject!
     {
-        return ["modified_time": "modifiedTime"]
+        return ["modified_time": "modifiedTime",
+                "subscriptions": "subscription"]
```

Comment:
Should the key here also be the singular "subscription," or do we support multiple subscriptions?